dashboards/memcached/memcached-prometheus.json

```diff
@@ -147,32 +173,42 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum(irate(memcached_read_bytes_total{${Cluster},${Location},${Namespace}}[5m]))"
+                  "prometheusQuery": "sum(irate(memcached_read_bytes_total{${Cluster},${Location},${Namespace}}[5m]))",
+                  "unitOverride": ""
                 }
               },
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "irate(memcached_written_bytes_total{${Cluster},${Location},${Namespace}}[10m])"
+                  "prometheusQuery": "irate(memcached_written_bytes_total{${Cluster},${Location},${Namespace}}[10m])",
```

Comment:
Looks like this already existed before, but can we make the read vs written bytes queries consistent? The read one (several lines above this) has a `sum()` but the written one doesn't -- so if there are multiple memcached instances in the same cluster, the former will aggregate all of them to produce a single time series, but the latter would produce one time series per memcached instance.

Either seems fine, but we should make it consistent.